rfcs/text/0000-data-source-categorization-fields.md

```diff
@@ -0,0 +1,75 @@
+# 0000: Data Source Categorization Fields
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
```

Comment:
I suggest we retarget to stage 1, since there's been so much discussion already.

Reply:
++